createlinks

```diff
@@ -49,6 +49,8 @@ event_templates('nethserver-sssd-save', qw(
     /etc/samba/smb.conf
     /etc/openldap/ldap.conf
     /etc/backup-config.d/nethserver-sssd.include
+    /etc/pam.d/cockpit
+    /etc/ssh/sshd_config
```

Comment:
Do not expand templates of dependent RPMs!

Let's to it the other way round: individual packages, when installed, will bind the `nethserver-sssd-save` event to expand _their own_ template and restart _their own_ service.

Reply:
ok, I am not sure to understand clearly, but let me advocate with another mind
What about if a sysadmin want to change the props `ShellOverrideStatus` to `enabled` manually, with only one evnt he can propagate his changes to any bound services ?

Reply:
Yes, the procedure would be

    config setprop sssd ShellOverrideStatus enabled
    signal-event nethserver-sssd-save

The point here is: the responsibility of expanding templates and restarting services is up to nethserver-cockpit and nethserver-openssh, so they have to plug in their templates and service restart into `nethserver-sssd-save` event from their own `createlinks` script.

Reply:
ok  it is clear, thank

Reply:
done